src/Core/Registration/GlobalOptimization.cpp

```diff
@@ -29,6 +29,8 @@
 #include <vector>
 #include <tuple>
 #include <Eigen/Dense>
+#include <Eigen/Sparse>
+#include <iostream>
```

Comment:
Do not use ``<iostream>``.
Use ``PrintInfo()`` and ``PrintDebug()``, see comments below.

Reply:
Will take care of this.

Reply:
But can you please explain why not <iostream> ?

Reply:
Using ``PrintDebug()`` and ``PrintInfo()`` etc will give the user an option to silence the printouts. Sometimes this is quite useful.